src/satellites.jl

```diff
@@ -31,16 +31,15 @@ end
 """
     gaussianKernel(x::Array{Float64, 1}, r::Float64)::Array{Float64, 1}
 
-Return Gaussian(x) for μ=0.0 and σ = r/3.
-(a value of σ = r/3 makes the training results comparable between different kernels
-for same values of r).
+Return Gaussian(x) for μ=0.0 and σ = r.
+
+(The gaussian is not "normal", but that makes it compare nicely with other kernels with r used as radius.)
 
 # Arguments
 
 """
-function gaussianKernel(x::Array{Float64, 1}, r::Float64)::Array{Float64, 1}
-
-    return Distributions.pdf.(Distributions.Normal(0.0,r/3), x)
```

Comment:
by removing Distributions.pdf I loose sensitivity. I don't see a significant performance issue. Run time is almost identical